### PR TITLE
Practice GitHub Flow

### PR DESCRIPTION
**Pull request by andrewchanintuit**
### TODO
- [x] demo markdown
- [x] demo @mentions
- [x] demo cross-links
- [ ] demo emoji

### CODE REVIEW

**Comment by andrewchanintuit** [comment]
@JosephineGitHub Your thoughts?

**Comment by andrewchanintuit** [comment]
If you haven't already, please comment on #2

**Comment by garyl553** [comment]
Hey Andrew, you're in this class too...

**Comment by andrewchanintuit** [comment]
Yes, hi @garyl553

**Comment by JosephineGitHub** [comment]
my thoughts are...zzz.

File: achan_hello_world.cc [hunk 1]
```diff
@@ -0,0 +1 @@
+where's the content?
```
Comment (garyl553) [inline achan_hello_world.cc:1]: Ok, much better.

**Pull request closed without merging**